### PR TITLE
Update Zapier to expose project status in addition to state

### PR DESCRIPTION
**Pull request by leelasn**
CleanShot 2024-03-11 at 17 41 16

### CODE REVIEW

**Review by mufeez-amjad — COMMENTED** [review]
should you also bump the package version?

**Comment by leelasn** [comment]
@mufeez-amjad provisionally bumped to 2.13.1 for now

File: src/triggers/project.ts [hunk 1]
```diff
@@ -39,12 +42,15 @@ const getProjectList = async (z: ZObject, bundle: Bundle) => {
...
-            filter: { state: { in: ["backlog", "started", "planned", "paused"] } }
+            filter: { status: { type: { in: ["backlog", "started", "planned"] } } }
           ) {
             nodes {
               id
               name
               state
+              status {
+                id
+              }
```
Comment (mufeez-amjad) [inline src/triggers/project.ts:53]: I think we should we include name and type

**Review by mufeez-amjad — APPROVED** [review]

File: src/triggers/project.ts [hunk 2]
```diff
@@ -39,12 +42,15 @@ const getProjectList = async (z: ZObject, bundle: Bundle) => {
             first: 50
             after: $after
             orderBy: updatedAt
-            filter: { state: { in: ["backlog", "started", "planned", "paused"] } }
+            filter: { status: { type: { in: ["backlog", "started", "planned"] } } }
```
Comment (urossmolnik) [inline src/triggers/project.ts:45]: Why removing paused?

**Review by urossmolnik — APPROVED** [review]

File: src/triggers/project.ts (hunk 2 above)
Comment (mufeez-amjad) [reply src/triggers/project.ts:45]: Will no longer be a valid `state` after LIN-19693

**Review by tommoor — APPROVED** [review]
Looks good, ~presumably this waits until backfill~

**Comment by leelasn** [comment]
@tommoor what backfill? We’ve backfilled and been double-writing to state and status for over a month

**Comment by tommoor** [comment]
I uploaded to Zapier development env, honestly `project.state` doesn't seem to be used anywhere that I can tell, it wouldn't hurt to have project status as a filter option for issues but currently this is essentially a no-op.

**Comment by leelasn** [comment]
@tommoor in that case let's just remove `state` from this trigger right now in addition to adding `status`?

**Comment by tommoor** [comment]
good plan, lets do that

**Pull request merged**